### PR TITLE
Updating to build images and build process

### PR DESCRIPTION
**Pull request by jasonsandlin**

### CODE REVIEW

**Review by MSFT-Heba — APPROVED** [review]

File: Build/libHttpClient.UnitTest.141.TE/libHttpClient.UnitTest.141.TE.vcxproj [hunk 1]
```diff
@@ -34,7 +34,7 @@
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <AdditionalOptions>/bigobj /Zm512 %(AdditionalOptions)</AdditionalOptions>
-      <AdditionalUsingDirectories>C:\Program Files (x86)\Windows Kits\10\UnionMetadata;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references;%(AdditionalUsingDirectories)</AdditionalUsingDirectories>
+      <AdditionalUsingDirectories>C:\Program Files (x86)\Windows Kits\10\UnionMetadata;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references;C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\vcpackages</AdditionalUsingDirectories>
```
Comment (jplafonta) [inline Build/libHttpClient.UnitTest.141.TE/libHttpClient.UnitTest.141.TE.vcxproj:37]: >C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\vcpackages [](http://example.com/codeflow?start=163&length=88)

intentional hard coded path? is there a variable that should be used here?

**Review by jplafonta — APPROVED** [review]
:shipit:

File: Build/libHttpClient.UnitTest.142.TE/libHttpClient.UnitTest.142.TE.vcxproj [hunk 2]
```diff
@@ -34,7 +34,7 @@
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <AdditionalOptions>/bigobj /Zm512 %(AdditionalOptions)</AdditionalOptions>
-      <AdditionalUsingDirectories>C:\Program Files (x86)\Windows Kits\10\UnionMetadata;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references;%(AdditionalUsingDirectories)</AdditionalUsingDirectories>
+      <AdditionalUsingDirectories>C:\Program Files (x86)\Windows Kits\10\UnionMetadata;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references;C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\VC\vcpackages;%(AdditionalUsingDirectories)</AdditionalUsingDirectories>
     </ClCompile>
```
Comment (jplafonta) [inline Build/libHttpClient.UnitTest.142.TE/libHttpClient.UnitTest.142.TE.vcxproj:38]: same comment here

File: Build/libHttpClient.UnitTest.141.TE/libHttpClient.UnitTest.141.TE.vcxproj (hunk 1 above)
Comment (jasonsandlin) [reply Build/libHttpClient.UnitTest.141.TE/libHttpClient.UnitTest.141.TE.vcxproj:37]: I don't see anything obvious in env vars.  Its just to enable build machine so using hardcoded path is fine as it doesn't matter if it doesn't exist on non-build machines

**Pull request merged**